cognite/powerops/bootstrap.py

```diff
@@ -167,22 +165,18 @@ def _transform(
     path: Path,
     market: str = "Dayahead",
 ) -> BootstrapResourceCollection:
-    cdf_parameters = config.cdf.dict()
-    client = get_client(cdf_parameters)
-    config = validate_config(config, client)
+    cognite_project = config.cdf.dict()["COGNITE_PROJECT"]
+    config = validate_config(config)
```

Comment:
Should we consider validating the config earlier in the code? For instance right after the config is loaded here;
https://github.com/cognitedata/power-ops-sdk/blob/main/cognite/powerops/main.py#L25

Reply:
yes, good idea